lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc

```diff
@@ -76,15 +79,14 @@ Again, circulate during the activity. As students match Verbal Expressions to th
 -  Fill in the empty spaces so that all three forms represent the same expression.
 }
 
-=== Synthesize
-
-Here are some key take-aways from this lesson:
+Here are some key take-aways:
 
 - We can find the @vocab{opposite} of an @vocab{absolute value}: @math{- | x |}
 - We can find the @vocab{absolute value} of an @vocab{opposite}: @math{ |- x |}
```

Comment:
Just noticing this now - the spacing on the negative sign is a little weird. Shouldn't it be `|-x|` and `-|x|`?